### ISSUE TITLE
Gear not "remembered" when switching jobs

### ISSUE THREAD

**Issue by Blex64** [EdenServer/community#401]
Not sure if this is a known issue! When switching jobs, all gear is removed, but the game doesn't appear to "remember" what gear you had on that job last. On retail, it will automatically equip any gear that you were wearing on that job that you still have on you.

**Comment by Siushaga** [comment]
I might not be remembering correctly, but I recall being naked after swapping "main" jobs in the old days too. An easy way around this is to create equipsets for each job you play and equip it after you switch job so you don't have to manually equip each and every piece after switching. Once you've created the equipset, you can name each one to keep track easier if you want and you equip it by command line "/equipset 1" or whatever set number it is. You can also add echo to the end of the command to have to display any problems with equipping.

**Comment by Siushaga** [comment]
Ah cool, glad I'm misremembering! Must be blurring the lines with so many private servers over the recent years. Haha

**Comment by godmode** [comment]
Yeah, easy to forget when era retail was 10 years ago and DSP was yesterday heh
